### PR TITLE
Add feature for supplier submits clarification question

### PR DESCRIPTION
**Pull request by robyoung**
[#113828715](https://www.pivotaltracker.com/story/show/113828715)

The functional tests for the [Supplier submits a clarification question](https://www.pivotaltracker.com/story/show/112340809) story.

### CODE REVIEW

File: features/supplier/supplier_submits_clarification_question.feature [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Feature: Supplier Submits Clarification Question
+
+Background: Publish a brief
```
Comment (TheDoubleK) [inline features/supplier/supplier_submits_clarification_question.feature]: `Publish requirements`?

File: features/supplier/supplier_submits_clarification_question.feature [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    Given I have a buyer user account
+    And I have deleted all draft briefs
+    And I have a 'published' brief
```
Comment (TheDoubleK) [inline features/supplier/supplier_submits_clarification_question.feature]: `I have a 'published' requirements`?

Comment (robyoung) [inline features/supplier/supplier_submits_clarification_question.feature]: `I have some 'published' requirements`?

File: features/step_definitions/setup_steps.rb [hunk 3]
```diff
@@ -279,7 +279,7 @@ def delete_all_draft_briefs (user_id)
   end
 end
 
-Given /^I have a '(.*)' brief$/ do |brief_state|
+Given /^I have a '(.*)' (?:brief|opportunity|set of requirements)$/ do |brief_state|
```
Comment (TheDoubleK) [inline features/step_definitions/setup_steps.rb]: It would be nice to purge `brief` and `briefs` altogether, but not necessarily for this PR.

**Comment by TheDoubleK** [comment]
:+1:

**Comment by robyoung** [comment]
:sparkles: this has been tested against preview :sparkles:

**Pull request merged**